### ISSUE TITLE
[Upgrade Request] PickPocket

### ISSUE THREAD

**Issue by coty9090** [peace-maker/smrpg#149]
PickPocket - Chance to steal 5 cash from your enemy when you knife them.

**Comment by peace-maker** [comment]
Couldn't this and #150 be merged or what do you intend to change when leveling up this upgrade in contrast to the #150 "Pro" version?

**Comment by coty9090** [comment]
Yeah, some upgrades are just about increasing the effectiveness of it's parent upgrade.
